pkg/agent/controller/egress/ipassigner/ip_assigner_linux.go

```diff
@@ -26,10 +25,10 @@ import (
 
 	"antrea.io/antrea/pkg/agent/util"
 	"antrea.io/antrea/pkg/agent/util/arping"
+	"antrea.io/antrea/pkg/agent/util/ndp"
+	ip2 "antrea.io/antrea/pkg/util/ip"
```

Comment:
ip2 is not a good alias. But I think you don't really need to import the package. See comment below.

Reply:
done.
Thanks for reviewing. Some comments have addressed. Still in progress, not completely processed.